src/index.html

```diff
@@ -1,13 +1,48 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card" data-qa="card">
+      <div class="card__monoblock-img"></div>
+
+      <h2 class="card__title-product">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+      <section class="card__product-code">Product code: 195434</section>
+
+      <section class="stars">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <section class="stars__reviews">Reviews: 5</section>
+      </section>
+
+      <section class="card__cost">
+        <div class="card__cost-price">Price:</div>
+        <div class="card__cost-value">$2,199</div>
+      </section>
+
+      <nav class="card__buy">
+        <a
+          class="card__buy-link"
+          href="#"
+          data-qa="hover"
+        >buy</a>
```

Comment:
```suggestion
        <a
          class="card__buy-link"
          href="#"
          data-qa="hover"
        >
          Buy
        </a>
```
pay attention to format (code style)